gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -255,7 +232,7 @@ public Builder toBuilder() {
     pb.setDescription(this.description());
     pb.setDnsName(this.dnsName());
     if (this.id() != null) {
-      pb.setId(this.id());
+      pb.setId(new BigInteger(this.id()));
```

Comment:
As said. Fine as is. Just FYI, the slight advantage that I see with my suggestion is that because
the Builder.id method is package scope a code outside of this class may supply a string which
is could not be converted to BigInteger.